CONTRIBUTION.md

```diff
@@ -1,42 +1,52 @@
 # Contribution guidlines
+
 ## Preamble
 These guidelines are intended as "guidelines" to how an PR To Dire's Buildinggadgets should be structured. 
 Bear in mind that these are only guidelines and don't necessarily apply in each and every case, but if you choose not to follow a specific Part of this specification, you must be able to reason out why, if asked to do so.
+
 ## PR structure
 ### General Notes
 In General a PR should explain what it does, how it does this and how it was tested. It should also follow the [Code Guidelines](#code-guidelines).
 * If you are not yet finished implementing, prepend it with \[WIP]
 * A description must be provided before the PR is merged, to provide record of the Performed changes
+
 ### Bug Fix PR's
 Your PR should include the following at least the following Information:
 * A reference to the corresponding issue or a full description of the issue being fixed
 * An explanation how this PR solves/attempts to solve the given issue
 * If testing was performed, what tests and how extensive they have been
+
 ### Feature Request PR's
 Your PR should include the following at least the following Information:
 * A reference to the corresponding issue or a full description of the feature being implemented
 * An explanation how this PR attempts to implement the given feature
 * If testing was performed, what tests and how extensive they have been
+
 ## Code Guidelines
 In General your code should be well-structured and easy to read. 
 We know that as of 1.12 this is not the case for great parts of the mod, but we are trying to improve upon the code-base.
-Also your PR should not contain any formatting only changes to Files, as this simply makes reviewing them unnecessarily complex.
-Try to avoid unnecessary commits in order not to pollute the commit history.
+Unless your Pr is directly related to code cleaning then your PR should not contain any formatting changes to unchanged code blocks as this simply makes reviewing them unnecessarily complex.
+
 ### Formatting
 * Use 4 space indent instead of Tabs
 * operators should be surrounded by spaces
-* a `,` comma to separate types should be followed by exactly one Space
+  * For example `if(statement == true)`
```

Comment:
Shouldn't there be a space in front of the `if` statement? Or we have not yet agreed on that one?

Reply:
Yes, a space after the `if` is definitely the established convention in the code base.

Reply:
I'm happy to include this as it is inline with these guidelines but I think if we're going to be aiming for code readbility then we should also add a space before the ) and after the ( 

So from this 
`if (statement == true)`
to 
`if ( statement == true )`

Reply:
If you ask me this has the opposite effect... @Phylogeny ?

Reply:
I'm also of the preference (aesthetic and otherwise) for no spaces.

Reply:
Eww, well, majority rules in favour of not me :P